writer_test.go

```diff
@@ -485,33 +496,37 @@ var writerTests = []struct {
 			&firstAndLastName{FirstName: "Leia", LastName: "Skywalker"},
 			&firstAndLastName{FirstName: "Luke", LastName: "Skywalker"},
 		},
-		dump: `row group 0
---------------------------------------------------------------------------------
-first_name:  BINARY ZSTD DO:4 FPO:55 SZ:90/72/0.80 VC:3 ENC:RLE_DICTIONARY,PLAIN ST:[min: Han, max: Luke, num_nulls not defined]
-last_name:   BINARY ZSTD DO:0 FPO:94 SZ:127/121/0.95 VC:3 ENC:DELTA_BYTE_ARRAY ST:[min: Skywalker, max: Solo, num_nulls not defined]
+		dump: `
+File path:  {file-path}
+Created by: github.com/parquet-go/parquet-go
+Properties: (none)
+Schema:
+message firstAndLastName {
+  required binary first_name (STRING);
+  required binary last_name (STRING);
+}
+
 
-    first_name TV=3 RL=0 DL=0 DS: 3 DE:PLAIN
-    ----------------------------------------------------------------------------
-    page 0:                        DLE:RLE RLE:RLE VLE:RLE_DICTIONARY ST:[no stats for this column] CRC:[PAGE CORRUPT] SZ:7 VC:3
+Row group 0:  count: 3  72.33 B records  start: 4  total(compressed): 217 B total(uncompressed):193 B
+--------------------------------------------------------------------------------
+            type      encodings count     avg size   nulls   min / max
+first_name  BINARY    Z _ R     3         30.00 B            "Han" / "Luke"
+last_name   BINARY    Z   D     3         42.33 B            "Skywalker" / "Skywalker"
```

Comment:
This looks like a bug in `(*writerColumn) recordPageStats` - this should be "Skywaker" / "Solo".

Reply:
The previous test case seemed like it rendered the correct data:

```
--------------------------------------------------------------------------------
first_name:  BINARY ZSTD DO:4 FPO:55 SZ:90/72/0.80 VC:3 ENC:RLE_DICTIONARY,PLAIN ST:[min: Han, max: Luke, num_nulls not defined]
last_name:   BINARY ZSTD DO:0 FPO:94 SZ:127/121/0.95 VC:3 ENC:DELTA_BYTE_ARRAY ST:[min: Skywalker, max: Solo, num_nulls not defined]
```

Definitely worth investigating before we merge.

Reply:
@achille-roussel Do we know that the old test case passes, or if not, when it may have stopped passing? From the test setup check, it doesn't look like these tests ran in CI.

Reply:
I did some further investigation and wrote up an issue for this. https://github.com/parquet-go/parquet-go/issues/53 From the look of it, it's an existing bug in (*writerColumn).recordPageStats that should have been caught by this test but presumably wasn't because nobody bothered to run this test?